### PR TITLE
Evaluate Arima script

### PR DESCRIPTION
**Pull request by TonichaC**

### CODE REVIEW

File: evaluate_arima.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	best_score, best_cfg = float("inf"), None
+	for p in p_values:
+		for d in d_values:
+			for q in q_values:
+				order = (p,d,q)
+				try:
+					mse = evaluate_arima_model(dataset, order)
+					if mse < best_score:
+						best_score, best_cfg = mse, order
+					print('ARIMA%s MSE=%.3f' % (order,mse))
```
Comment (hstewart93) [inline evaluate_arima.py]: You could use f strings here, its the most up to date string formatting in python 3 and is often easier to read :) https://realpython.com/python-f-strings/

File: evaluate_arima.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+				except:
+					continue
+	print('Best ARIMA%s MSE=%.3f' % (best_cfg, best_score))
```
Comment (hstewart93) [inline evaluate_arima.py]: Same here with f strings

File: evaluate_arima.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+df = df.drop("Date",axis=1)
+
+# evaluate an ARIMA model for a given order (p,d,q)
+def evaluate_arima_model(X, arima_order):
+	# prepare training dataset
+	train_size = int(len(X) * fraction_training)
+	train, test = X[0:train_size], X[train_size:]
+	history = [x for x in train]
+	# make predictions
+	predictions = list()
```
Comment (hstewart93) [inline evaluate_arima.py]: Can also be written as `predictions = []`

File: evaluate_arima.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import warnings
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+from statsmodels.tsa.arima_model import ARIMA
+from sklearn.metrics import mean_squared_error
```
Comment (hstewart93) [inline evaluate_arima.py]: Generally, imports should be in alphabetical order, if you install pre-commit hooks (instructions in README) it should alert you to linting errors.

File: evaluate_arima.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+from get_data.get_data import call_api
```
Comment (hstewart93) [inline evaluate_arima.py]: 👍 The linting here is correct, there should be a blank line between third party imports and application module imports :)

**Review by hstewart93 — COMMENTED** [review]
Please could you create a directory to store these sorts of files in and move this file inside. Also please set up pre-commit hooks in your virtual environment and it should fix any formatting issues :)

**Review by mrknorman — APPROVED** [review]
Looks good to me, although you could use fstrings (see me and Hattie's extensive back and forth), just for neatness, and I don't think you ran it through the autoformatter

**Pull request merged**